x-pack/plugins/index_management/server/routes/api/indices/fetch_aliases.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export default async function fetchAliases(callWithRequest) {
+  const catAliases = await callWithRequest('cat.aliases', { format: 'json' });
```

Comment:
Let's switch this to `indices.getAlias({ index: '*', allowNoIndices: true })` so it works with a lower security level, pretty sure the `cat.*` APIs require cluster-level permissions that many users won't have.

Reply:
K, makes sense did not know there was another way to get the aliases for a index.